src/lib/gui/dialogs/FingerprintDialog.cpp

```diff
@@ -125,8 +125,8 @@ void FingerprintDialog::togglePreviewMode(bool hashMode)
 
 void FingerprintDialog::updateModeButton(bool hashMode) const
 {
-  const auto text = tr("Show %1").arg(hashMode ? "Image" : "Hash");
-  const auto toolTip = tr("Display the fingerprint as %1").arg(hashMode ? "an image" : "a hash");
+  const auto text = tr("Show %1").arg(hashMode ? tr("Image") : tr("Hash"));
```

Comment:
Same as last comment.

```suggestion
  const auto text = hashMode ? tr("Show image") : tr("Show hash");
```